evergreen/eg_app/views.py

```diff
@@ -265,13 +267,68 @@ def uploadPost(request) -> JsonResponse:
             post = Post.objects.create(user=user, image=image, caption=caption)
             post.save()
 
+            # USES web sockets to braodcast to ALL CLIENTS
```

Comment:
some of yall dont have spell check in your ide and it shows
```suggestion
            # USES web sockets to broadcast to ALL CLIENTS
```